src/e2e-test/java/io/cdap/plugin/datastore/stepsdesign/StepDesigns.java

```diff
@@ -0,0 +1,17 @@
+package io.cdap.plugin.datastore.stepsdesign;
```

Comment:
Also add separate folder structure for actions, locators, stepdesign and place the respective class files in each folder. refer folder structuring of any existing plugin.